spring-cloud-streams-samples/source/src/main/resources/application.yml

```diff
@@ -4,7 +4,7 @@ fixedDelay: 5000
 spring:
   cloud:
     channels:
-      inputChannelName: testtock
+      outputChannelName: testtock
```

Comment:
This looks like a slip. We haven't been supporting this configuration model since some time.